script/create_release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+REPO_NAME=kubeless
```

Comment:
just remember that this was here, can you move these env var to the circle-ci file? (so we can reuse the script for the different triggers)